app/UiTPASService/UiTPASServiceEventControllerProvider.php

```diff
@@ -26,44 +29,49 @@ public function connect(Application $app): ControllerCollection
             )
         );
 
-        $app['uitpas.event_card_systems_controller'] = $app->share(
-            function (Application $app) {
-                return new EventCardSystemsController(
-                    $app['uitpas']
-                );
-            }
+        $app[GetCardSystemsFromEventRequestHandler::class] = $app->share(
+            fn (Application $app) => new GetCardSystemsFromEventRequestHandler(
+                $app['uitpas']
```

Comment:
Nit: The constructors of these request handlers could be on one line since it's only argument